setup.py

```diff
@@ -27,7 +27,7 @@
 
 AVRO_REQUIRES = ['fastavro>=0.23.0,<1.0;python_version<"3.0"',
                  'fastavro>=1.0;python_version>"3.0"',
-                 'avro==1.10.0'
+                 'avro>=1.11.1,<2',
```

Comment:
Also need to update tests/requirements.txt and src/confluent_kafka/avro/requirements.txt